content/docs/querying/basics.md

```diff
@@ -85,7 +85,7 @@ For example, this selects all `http_requests_total` time series for `staging`,
     http_requests_total{environment=~"staging|testing|development",method!="GET"}
 
 Label matchers that match empty label values also select all time series that do
-not have the specific label set at all. Regex-matches are fully anchored.
+not have the specific label set at all. Regex-matches are not anchored.
```

Comment:
I'd take partial match to mean that only some of the regex matched, not that only some of the text matched.